### PR TITLE
OCPEDGE-1091: adding reporting for ibu periodics

### PR DESCRIPTION
**Pull request by jeff-roche**

### CODE REVIEW

**Comment by jeff-roche** [comment]
/pj-rehearse periodic-ci-openshift-kni-lifecycle-agent-main-nightly-4.16-image-based-install periodic-ci-openshift-kni-lifecycle-agent-main-nightly-4.16-image-based-upgrade-e2e-serial-conformance

File: ci-operator/jobs/openshift-kni/lifecycle-agent/openshift-kni-lifecycle-agent-main-periodics.yaml [hunk 1]
```diff
@@ -18,6 +18,14 @@ periodics:
     ci.openshift.io/generator: prowgen
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
   name: periodic-ci-openshift-kni-lifecycle-agent-main-nightly-4.15-image-based-upgrade-e2e-parallel-conformance
+  reporter_config:
+    slack:
+      channel: '#image-based-ci'
+      job_states_to_report:
+      - failure
+      - success
```
Comment (suleymanakbas91) [inline ci-operator/jobs/openshift-kni/lifecycle-agent/openshift-kni-lifecycle-agent-main-periodics.yaml:26]: Why do we want to be reported for the successful runs?

Comment (jeff-roche) [reply ci-operator/jobs/openshift-kni/lifecycle-agent/openshift-kni-lifecycle-agent-main-periodics.yaml:26]: I plan to remove this in the future, a success is basically non-existant at the moment

**Review by omertuc — CHANGES_REQUESTED** [review]
Let's also add [aborted](https://github.com/openshift/ci-tools/blob/d60196771ee16276a30ace000252e4821cae3a2a/vendor/k8s.io/test-infra/prow/apis/prowjobs/v1/types.go#L67) to all of them?

**Comment by eggfoobar** [comment]
Looks good, removing hold now that `aborted` has been added.

/unhold
/lgtm

**Pull request merged**